configs/ext/README.md

```diff
@@ -0,0 +1,20 @@
+Summary
+-------
+
+All files in this directory are distributed under the normal Mbed TLS dual Apache 2.0 or GPLv2-or-later
+license, and are copyright The Mbed TLS Contributors.
```

Comment:
Given this, and the commentary in `Background` should we maybe rename this directory to `tfm`, since these licensing statements are unlikely to be true if we ever bring in other third-party licences, and we may forget at that point

Reply:
Yes, that might be better - thanks